composeml/label_times/plots.py

```diff
@@ -28,6 +28,7 @@ def count_by_time(self, ax=None, **kwargs):
         """Plots the label distribution across cutoff times."""
         count_by_time = self._label_times.count_by_time
         count_by_time.sort_index(inplace=True)
+        target_column = self._label_times.target_columns[0]
```

Comment:
Do we need to assert that there is a single target first?

Reply:
Upon further review, I guess this isn't necessary as this gets checked in `LabelTimes.count_by_time()`.